test/Microsoft.Data.Analysis.Tests/DataFrameTests.cs

```diff
@@ -1658,7 +1658,7 @@ public void TestMerge()
             Assert.Equal(16, merge.Rows.Count);
             Assert.Equal(merge.Columns.Count, left.Columns.Count + right.Columns.Count);
             Assert.Null(merge.Columns["Int_left"][12]);
-            Assert.Null(merge.Columns["Int_left"][5]);
+            Assert.Null(merge.Columns["Int_left"][15]);
```

Comment:
`5` -> `15` because we don't preserve the ordering of rows in `OuterJoin`. I don't think this needs to be a requirement. If somebody requests it at some point, we can think about preserving the ordering then.